app/src/main/java/org/mozilla/tv/firefox/fxa/ADMIntegration.kt

```diff
@@ -40,12 +49,18 @@ class ADMIntegration(private val app: Application) {
         AutoPushFeature(app, ADMService(), PushConfig(senderId = senderId, serviceType = ServiceType.ADM))
     }
 
-    fun createSendTabFeature(accountManager: FxaAccountManager) {
+    private val _receivedTabsRaw: PublishSubject<ReceivedTabs> = PublishSubject.create()
+    val receivedTabsRaw: Observable<ReceivedTabs> = _receivedTabsRaw.hide()
+
+    fun createSendTabFeature(
+        accountManager: FxaAccountManager,
+        @VisibleForTesting(otherwise = NONE) pushFeature: AutoPushFeature = this.pushFeature
```

Comment:
I found a startup crash on emulator that I suspect may be coming from this line

Reply:
`pushFeature` is trying to be created on `init` of `FxaRepo`, but ADM class does not exist.

Reply:
This is the same failure that is causing the UI tests and I just forced pushed a version without my tests + refactor that caused this error.